File: src/main/java/com/devmpv/exceptions/CRException.java
package com.devmpv.exceptions;

/**
 * Chan-reactor exception
 * 
 * @author devmpv
 *
 */
public class CRException extends RuntimeException {

    /**
     * 
     */
    private static final long serialVersionUID = 5748872141056604785L;

    public CRException(String message) {
	super(message);
    }

    public CRException(String message, Throwable cause) {
	super(message, cause);
    }

}

File: src/main/kotlin/com/devmpv/exceptions/CRException.kt
package com.devmpv.exceptions

/**
 * Chan-reactor exception
 *
 * @author devmpv
 */
class CRException : RuntimeException {

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    companion object {

        private val serialVersionUID = 5748872141056604785L
    }

}
